db/models.py

```diff
@@ -14,3 +14,33 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
```

Comment:
Avoid overusing f-strings when they are not necessary. According to the checklist, you should return the string directly without using f-string formatting.